content/03-reference/01-tools-and-interfaces/01-prisma-schema/06-relations.mdx

```diff
@@ -888,58 +895,89 @@ Note that self-relations always require the `@relation` attribute.
 
 ### One-to-one self-relations
 
-A one-to-one self-relation looks as follows:
+The following example models a one-to-one self-relation:
 
 ```prisma
 model User {
-  id       Int      @id @default(autoincrement())
-  name     String?
-  husband  User      @relation("MarriagePartners", fields: [id], references: [id])
-  wife     User?     @relation("MarriagePartners")
+  id             Int     @default(autoincrement()) @id
+  name           String?
+  successorId Int?
+  successor   User?   @relation("BlogOwnerHistory", fields: [successorId], references: [id])
+  predecessor User?   @relation("BlogOwnerHistory")
 }
 ```
 
 This relation expresses the following:
 
-- "a user has zero or one _husband_ "
-- "a user can have zero or one _wife_"
+- "a user can have one or zero predecessors" (for example, Sarah is Mary's predecessor as blog owner)
+- "a user can have one or zero successors" (for example, Mary is Sarah's successor as blog owner)
 
-Note that one-to-one self-relations can not be made required on both sides, one field has to be optional.
+> **Note**: One-to-one self-relations cannot be made required on both sides. One or both sides must be optional, otherwise it becomes impossible to create the first `User` record.
 
-Self-relations always require the `@relation` attribute on both sides of the relation. On both ends, you need provide the same relation _name_.
+To create a one-to-one self-relation:
 
-The `husband` field on the other hand is the [annotated relation field](#annotated-relation-fields-and-relation-scalar-fields) (although here the term is ambiguous because both relation fields are annotated with the `@relation` attribute, however here this refers too the field that also has the `fields` and `references` arguments).
+- Both sides of the relation must define a `@relation` attribute that share the same name - in this case, **BlogOwnerHistory**.
+- One relation field must be a [fully annotated](#annotated-relation-fields-and-relation-scalar-fields). In this example, the `successor` field defines both the `field` and `references` arguments.
+- One relation field must be backed by a foreign key. The `successor` field is backed by the `successorId` foreign key, which references a value in the `id` field.
```

Comment:
the bit about the database (old line 913) is important here in my opinion, as this sentence otherwise mixes terminology of Prisma and database

Reply:
"One relation field must be backed by a foreign key **in the underlying database**"?

Reply:
Much better, yes.